src/StochOptInterface.jl

```diff
@@ -1,9 +1,14 @@
-# TODO This module should be replaced by the package
-# https://github.com/JuliaStochOpt/StochOptInterface.jl
+#  Copyright 2018, JuliaStochOpt and contributors.
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+################################################################################
+# StochOptInterface
+# A Julia package to encode Stochastic Programming problems.
+################################################################################
 module StochOptInterface
 
-using DocStringExtensions
```

Comment:
This causes Travis to fail at the moment

Reply:
My bad ...